base/broadcast.jl

```diff
@@ -1216,6 +1216,7 @@ Base.@propagate_inbounds dotview(args...) = Base.maybeview(args...)
 # broadcasting "dot" calls/assignments:
 
 dottable(x) = false # avoid dotting spliced objects (e.g. view calls inserted by @view)
```

Comment:
Wait, this seems to conflict --- the view macro depends on not dotting the spliced `view` function?

Reply:
Yeah, I'm still confused about this point, too.  The `@view` macro still works with this definition, and I _think_ it's because embeds a bigger `(true && view(...))` expr.  But maybe the default should be `dottable(x) = true` to support types and more?  Perhaps it predates moving the _literal_ interpolation check higher up?

Reply:
Yeah, that's why I used `nameof` before.

Reply:
Oh, apparently it doesn't work in all situations — the test errors are related to `@view`.  I had just done some basic tests at the REPL.

Reply:
`dottable(:view)` is true though, so `nameof` doesn't fix this.